.changeset/nine-games-clap.md

```diff
@@ -0,0 +1,7 @@
+---
+"@neo4j/graphql": patch
+---
+
+
+Deprecate relationship filtering using the non-generic version such as `actors_SOME:{ title_EQ: "The Matrix" }` in favor of the dedicated input `actors: { some: { title: { eq: "The Matrix" } } }`.
```

Comment:
```suggestion
Deprecate relationship filtering using the non-generic version such as `actors_SOME: { title_EQ: "The Matrix" }` in favor of the dedicated input `actors: { some: { title: { eq: "The Matrix" } } }`.
```

Reply:
I see the terms "non-generic" and "dedicated" both used here. Is that intentional?

Reply:
Good point, not there was no specific reasoning behind!